_posts/2018-06-29-14-04-netcore-7-0.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title : "14.4 OData Web API 7.0 now available"
+description: "7.0 release notes"
+category: "14. 7.x Features "
+---
+
+Today we're happy to announce the release of ASP.NET Web API OData 7.0 on the [NuGet gallery](https://www.nuget.org/)!
+
+ASP.NET Web API OData 7.0 is available with two new packages:
+
+1. [Microsoft.AspNetCore.OData](https://www.nuget.org/packages/Microsoft.AspNetCore.OData) is based on ASP.NET Core 2.x.
+2. [Microsoft.AspNet.OData](https://www.nuget.org/packages/Microsoft.AspNet.OData) is based on ASP.NET Web API.
+
+[Here](http://odata.github.io/WebApi/#14-01-netcore-beta1) can find beta1, beta2 & beta4 information.
+
```

Comment:
I think we don't need beta pointers in RTM.

Reply:
Re-added.